FrontEnd/src/router/index.js

```diff
@@ -24,6 +24,14 @@ const router = createRouter({
       // which is lazy-loaded when the route is visited.
       component: () => import("../views/AddNgoView.vue"),
     },
+    {
+      path: "/kidsCheckUp",
+      name:"kidsCheckUp",
+      // route level code-splitting
```

Comment:
These comments can be removed, I have kept early because any one in future will refer and know why we have used so we don't need to repeat the same comment here.